SwiftCompilerSources/Sources/Optimizer/FunctionPasses/LifetimeDependenceDiagnostics.swift

```diff
@@ -29,6 +29,11 @@ private func log(prefix: Bool = true, _ message: @autoclosure () -> String) {
 let lifetimeDependenceDiagnosticsPass = FunctionPass(
   name: "lifetime-dependence-diagnostics")
 { (function: Function, context: FunctionPassContext) in
+#if os(Windows)
```

Comment:
Can we consider limiting this to `os(Windows) && arch(arm64)` and same throughout? Is there any thing that would be useful that we could gather to help isolate this issue?

Reply:
> Can we consider limiting this to `os(Windows) && arch(arm64)` and same throughout?

Not much point. These passes literally do nothing at the moment. They just process the SIL using a few SwiftCompilerSources helpers and crash if the SIL is invalid according to those helpers.

> Is there any thing that would be useful that we could gather to help isolate this issue?

For the crashing function, identifying the difference in -emit-silgen between x86 and arm64 would be a good start. I can't imagine why there's any difference. But since I won't be able to look at it for a week, there's no reason to hold up any toolchain builds.

Reply:
Okay, sounds like the value of the additional SIL validation is low?